graphqlapi/src/main/java/io/stargate/graphql/graphqlservlet/CassandraUnboxingGraphqlErrorHandler.java

```diff
@@ -23,6 +23,7 @@
 import java.util.List;
 
 public class CassandraUnboxingGraphqlErrorHandler implements GraphQLErrorHandler {
```

Comment:
I know this is an existing name, but "unboxing" is a confusing name because it's more "processing" the errors or "converting" them. What about changing this to `CassandraGraphqlErrorHandler` and changing "unbox" to "convert" or "process".

Reply:
Makes sense to give it another name, how `StargateGraphqlErrorHandler` sounds?

Reply:
I've changed it to `StargateGraphqlErrorHandler`, lmkwyt